### ISSUE TITLE
关于tf.nn.conv2d_transpose接口的调用问题

### ISSUE THREAD

**Issue by huiyang865** [yindaheng98/GANomaly-Tensorflow#1]
你好，感谢分享代码。

最近正在调试您的代码，发现出现了一个奇怪的问题，我在运行网络时，在反卷积部分tf会调用Conv2DBackpropInput[T=DT_FLOAT, data_format="NCHW", dilations=[1, 1, 1, 1], padding="VALID", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, ...)接口。

调用代码为：conv = tf.nn.conv2d_transpose(x, filter=w_conv, strides=[1, 1, 1, 1], padding='VALID', data_format='NHWC', output_shape=[-1, shape, shape, channel])   这里data_format='NHWC'已经限定了数据的格式，但是tf内部代码还是会调用数据格式为"NCHW"的相应代码，并且报错如下：
InvalidArgumentError (see above for traceback): Dimension -1 must be >= 0
	 [[{{node decoder1/conv2d_transpose}} = Conv2DBackpropInput[T=DT_FLOAT, data_format="NCHW", dilations=[1, 1, 1, 1], padding="VALID", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:GPU:0"](decoder1/conv2d_transpose-0-LayoutOptimizer, decoder1/decoder_wconv_initial/read, encoder1/Conv2D_6)]]
	 [[{{node add_1/_13}} = _Recv      [client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_3099_add_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

请问你有遇到这个问题么？方便的话可以告诉我你的tensorflow版本么？谢谢

**Comment by yindaheng98** [comment]
我好像没有遇见这个问题
我的tenserflow版本是1.10
使用的轮子来自[https://github.com/fo40225/tensorflow-windows-wheel](https://github.com/fo40225/tensorflow-windows-wheel)

**Comment by Jverson** [comment]
@huiyang865 我最近在使用tf.nn.conv2d_transpose时也遇到了这个问题，明明设的是NHWC，却报错并显示data_format='NCHW'
请问你之前是怎么解决这个问题的？
